src/blocks/battery.rs

```diff
@@ -31,6 +32,10 @@ pub struct BatteryConfig {
     /// Which BAT device in /sys/class/power_supply/ to read from.
     #[serde(default = "BatteryConfig::default_device")]
     pub device: String,
+
+    /// Show only percentage, time until (dis)charged or both
+    #[serde(default = "BatteryConfig::default_show")]
+    pub show: String,
```

Comment:
Can you implement this as an enum, so it can be handled directly by `serde`? There are some examples of this elsewhere in the code, including [this one](https://github.com/greshake/i3status-rust/blob/master/src/blocks/disk_space.rs#L38-L46). Using an enum would also remove the need to handle the `_` case in the match block below, since it would error out (with a helpful message) when reading the config file, instead.

Reply:
Good point, thanks!